### PR TITLE
bring back text generation webui but branch on repo idk what the fuck i do with the original pr

### PR DESCRIPTION
**Pull request by Butterroach**

### CODE REVIEW

**Comment by Butterroach** [comment]
btw on the client side just put "textgenwui" in data as anything

**Review by artifishvr — APPROVED** [review]
did butterroach "cook" as the kids say?

**Comment by Butterroach** [comment]
Restore branch

**Comment by artifishvr** [comment]
@spongedsc/web kill

File: .env.example [hunk 1]
```diff
@@ -1 +1,3 @@
-CHARACTERAI_TOKEN=
\ No newline at end of file
+CHARACTERAI_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+TEXTGENUI_ENDPOINT=http://127.0.0.1:5000/v1/chat/completions
```
Comment (Butterroach) [inline .env.example:2]: no w in textgenui inconsistency i am going to open a new issue

Comment (artifishvr) [reply .env.example:2]: you should have left this before i merged

Comment (Butterroach) [reply .env.example:2]: i only checked the commits after you merged

Comment (artifishvr) [reply .env.example:2]: bad team-member

Comment (Butterroach) [reply .env.example:2]: :(

**Pull request merged**